webapp/views.py

```diff
@@ -0,0 +1,73 @@
+from flask import render_template
```

Comment:
the goal of all those views in the future is to move them to ubuntu.com. Could you create a blueprint for them so it's easier to move it from a project to another?

Reply:
sure  :)

Reply:
@tbille updated it